runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineOptions.java

```diff
@@ -141,6 +141,13 @@
 
   void setDataflowWorkerJar(String dataflowWorkerJar);
 
+  @Description(
```

Comment:
Will this eventually become the default behavior? Is there any reason to support both?

If this will become default, I suggest using an [experiment](https://github.com/apache/beam/blob/847f3e74f35bdd2163ed4fa00b31a8f6b3c0f673/sdks/java/core/src/main/java/org/apache/beam/sdk/options/ExperimentalOptions.java#L37) pipeline option rather than a named option. PipelineOptions become part of the public API and are harder to drop later.

Reply:
Done.